### ISSUE TITLE
css and js first loading problem

### ISSUE THREAD

**Issue by srdrsvml1986** [nopSolutions/nopCommerce#7166]
nopCommerce version: 4.7

Steps to reproduce the problem: 
![image](https://github.com/nopSolutions/nopCommerce/assets/12673971/0546f0c9-86d9-4509-8377-03111959e669)

the problem is:
When any page is loaded for the first time, I get an error like this. The screenshot is also shared as an attachment.

**Refused to apply style from 'https://elacigdem.com/css/ui6y-pk2ee-opru-2vpxnq.styles.css?v=S89BItUIc3NwjJiwcd7e-zXXmTY' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.

Refused to execute script from 'https://elacigdem.com/js/public.estimateshipping.popup.js?v=LNAFBYlZhrhKSy8fRXUlcnk_yzM' because its MIME type ('text/html') is not executable, and strict MIME type checking is enabled.**

In other words, the requested css and js files cannot be analyzed because they are detected in text format. However, this is resolved when I click the refresh button on the same page.

**Comment by RomanovM** [comment]
@srdrsvml1986 Check the `"CacheDirectory"` setting in appsettings.json, there should be a value like `"~/wwwroot/bundles"`. 
You can also search for a solution on [our forums](https://www.nopcommerce.com/boards/).
